bin/managed-component-to-cloudflare-worker.js

```diff
@@ -134,9 +254,19 @@ WORKER_NAME: Name of the Cloudflare Worker to be created `)
   const userInput = await prompt(
     `\nYour Cloudflare Worker will be named "${workerName}".\nEnter "yes" to deploy with Wrangler: `
   )
-
   if (userInput !== 'yes') exit(0)
```

Comment:
let's use isYesResponse also here, because it's confusing that I can type `y` in some questions but here I _have_ to type `yes`